server/app/models/user.rb

```diff
@@ -40,6 +40,7 @@ def get_advert_count
   end
 
   def jwt_payload
-    { "email" => self.email, "id" => self.id }
+    @role = Role.find(self.role_id)
+    { "email" => self.email, "id" => self.id, "role" => @role.name }
```

Comment:
it could be `self.role.name`, or `self.role_name` if you add delegation of name method to role association in the beginning of User model, e.g:

```
  delegate :name, to: :role, prefix: true
```
ref: https://api.rubyonrails.org/classes/Module.html#method-i-delegate
<img width="983" alt="Screenshot 2022-04-13 at 07 46 53" src="https://user-images.githubusercontent.com/201830/163102187-38b8c51d-9986-4de7-a29a-6de0e52fd461.png">

Reply:
changes will be taken from feature/isuue_26 pull-request